mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.kt

```diff
@@ -524,6 +527,85 @@ class MainActivity : AbstractBaseActivity(), ConnectionFactory.UpdateListener {
         }
     }
 
+    private fun selectServerBasedOnWifi() {
+        fun getPermissionForSsidRead() = when {
+            Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q -> Manifest.permission.ACCESS_FINE_LOCATION
+            Build.VERSION.SDK_INT >= Build.VERSION_CODES.O -> Manifest.permission.ACCESS_COARSE_LOCATION
+            else -> null
+        }
+
+        val serverHaveWifiSet = prefs
```

Comment:
I would suggest to prefix this with "does"

Reply:
What about `hasAnyServerWifiSet`?

Reply:
`anyServerHasSetWifi`? (Putting `Set` in the middle to not imply this variable actually is a set)

Reply:
Sounds reasonable